engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/WorkflowExecutionActorData.scala

```diff
@@ -43,19 +43,13 @@ case class WorkflowExecutionActorData(workflowDescriptor: EngineWorkflowDescript
   /** Checks if the workflow is completed by scanning through the executionStore.
     * If complete, this will return Some(finalStatus).  Otherwise, returns None */
   def workflowCompletionStatus: Option[ExecutionStatus] = {
+    def upstream(scope: Scope): List[Scope] = scope.prerequisiteScopes.toList ++ scope.prerequisiteScopes.toList.flatMap(upstream)
```

Comment:
ToL: In this spirit of "this will all be different in a month", I believe this is part of wdl4s++